tests/testing.h

```diff
@@ -36,7 +36,7 @@
 #include <sstream>
 #include <functional>
 
-#ifdef _WIN32
+#if (defined _WIN32) || (defined SDL2PP_XCODE_GENERATED)
```

Comment:
Isn't it possible to detect xcode right away without introducing extra cmake variable?